.travis.yml

```diff
@@ -1,4 +1,10 @@
+os:
+  - linux
+branches:
+  only:
+    - master
```

Comment:
no PRs are always tested

if you commit to an open PR travis re-runs each commit

but let's say you were working on a branch and wanted to save some work... if you haven't PRed that yet then travis won't bother running and showing a bunch of extra test outcomes in the travis build area on their site, so we can suppress noise from builds we're not interested in looking at